### DISCUSSION TITLE
Questão TI21#10

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#17] [Q&A] [answered]
Explique o conceito de "polimorfismo" em C#.

**Comment by Guilhlm** [comment]
Polimorfismo na orientação a objetos você pode enviar uma mesma mensagem para diferentes objetos e fazê-los responder da maneira correta

**Comment by RenonHG** [comment]
Polimorfismo, na programação ao todo não só no C#, é a capacidade de um objeto ser tratado como a instância de várias classes ao mesmo tempo.

**Comment by Malvadaokkj** [comment]
É a propriedade de duas ou mais classes de uma mesma superclasse responderem a mesma mensagem, só que sendo cada uma de forma diferente

**Comment by Arrowaaa** [comment, marked as answer]
É a capacidade de um objeto ser tratado como a instância de várias classes ao mesmo tempo.
